src/ontology/ro-edit.owl

```diff
@@ -6147,6 +6149,17 @@ AnnotationAssertion(rdfs:label obo:RO_0015003 "subcluster of")
 SubObjectPropertyOf(obo:RO_0015003 obo:BFO_0000050)
 TransitiveObjectProperty(obo:RO_0015003)
 
+# Object Property: obo:RO_0015004 (has characterizing marker set)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0015004 "transcriptomically defined cell type X subClass_of: (has_characterizing_marker_set some S1); S1 has_part some gene 1, S1 has_part some gene 2, S1 has_part some gene 3, S1 has_anatomical_context some [region]")
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0015004 "A relation between a material entity and a set of markers that indentifies it in a particular context.")
```

Comment:
Suggested def:

A relation that applies between a cell type and a set of markers that can be used to uniquely identify that cell type.